### PR TITLE
[ML] Add correlation tooltip

### PR DESCRIPTION
**Pull request by lcawl**
## Summary

Adds a tooltip for the new Correlation column in https://github.com/elastic/kibana/pull/99905

### Screenshot

image

### CODE REVIEW

**Review by benwtrent — APPROVED** [review]

File: x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx [hunk 1]
```diff
@@ -229,9 +230,31 @@ export function MlLatencyCorrelations({ onClose }: Props) {
...
-        name: i18n.translate(
-          'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationLabel',
-          { defaultMessage: 'Correlation' }
+        name: (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationColumnDescription',
+              {
+                defaultMessage:
+                  'The impact of a field on the latency of the service, ranging from 0 to 1.',
```
Comment (tveasey) [inline x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx:239]: I might add one sentence which is something like...
```suggestion
                  'The impact of a field on the latency of the service, ranging from 0 to 1. The larger the value the more the field increases latency.',
```

**Review by tveasey — APPROVED** [review]
Just one minor suggestion to say something about larger values mean larger increase in latency.

File: x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx (hunk 1 above)
Comment (lcawl) [reply x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx:239]: Thanks for the feedback!  This tooltip was already merged into https://github.com/elastic/kibana/pull/99905, so I've amended the text in https://github.com/elastic/kibana/pull/103759

**Comment by lcawl** [comment]
These changes were implemented via 540d206e882833c71ae15a88ee1f52425413dada in https://github.com/elastic/kibana/pull/99905

**Pull request closed without merging**